ui/raidboss/data/07-dt/raid/r3s.ts

```diff
@@ -252,6 +302,198 @@ const triggerSet: TriggerSet<Data> = {
+          dir: cloneDir,
+        });
+        if (data.phaseTracker === 1 && data.tagTeamClones.length === 2)
+          return true;
+        return false;
+      },
+      infoText: (data, _matches, output) => {
+        const myClone = data.tagTeamClones.find((clone) => clone.dir === data.tagTeamCloneTethered);
+        const otherClone = data.tagTeamClones.find((clone) =>
+          clone.dir !== data.tagTeamCloneTethered
+        );
+
+        if (myClone === undefined) {
+          console.error('R3S Tag Team Clone: Missing myClone', data.tagTeamClones);
+          return;
+        }
+
+        if (otherClone === undefined) {
+          console.error('R3S Tag Team Clone: Missing otherClone', data.tagTeamClones);
+          return;
+        }
+
+        const [safeSpots, lastSafeSpot] = getSafeSpotsFromClones(myClone, otherClone);
+
+        return output.safeDirs!({
+          dirs: safeSpots
+            .map((dir) => output[Directions.outputFrom8DirNum(dir)]!())
+            .join(output.separator!()),
+          last: output[Directions.outputFrom8DirNum(lastSafeSpot)]!(),
+        });
+      },
+      run: (data) => {
+        data.tagTeamCloneTethered = undefined;
+        data.tagTeamClones = [];
+      },
+      outputStrings: tagTeamOutputStrings,
+    },
+    {
+      id: 'R3S Tag Team Murderous Mist',
+      type: 'StartsUsingExtra',
+      netRegex: { id: '9BD7', capture: true },
+      // Sometimes this MM cast is before the clones, sometimes it's after
+      delaySeconds: 0.1,
+      infoText: (data, matches, output) => {
+        const myClone = data.tagTeamClones.find((clone) => clone.dir === data.tagTeamCloneTethered);
+        const otherClone = data.tagTeamClones.find((clone) =>
+          clone.dir !== data.tagTeamCloneTethered
+        );
+
+        if (myClone === undefined) {
+          console.error('R3S Tag Team Clone: Missing myClone', data.tagTeamClones);
+          return;
+        }
+
+        if (otherClone === undefined) {
+          console.error('R3S Tag Team Clone: Missing otherClone', data.tagTeamClones);
+          return;
+        }
+
+        const murderousMistDir = Directions.hdgTo8DirNum(parseFloat(matches.heading));
+
+        const [safeSpots, lastSafeSpot] = getSafeSpotsFromClones(
+          myClone,
+          otherClone,
+          murderousMistDir,
+        );
+
+        return output.safeDirs!({
+          dirs: safeSpots
+            .map((dir) => output[Directions.outputFrom8DirNum(dir)]!())
+            .join(output.separator!()),
+          last: output[Directions.outputFrom8DirNum(lastSafeSpot)]!(),
+        });
+      },
+      run: (data) => {
+        data.tagTeamCloneTethered = undefined;
+        data.tagTeamClones = [];
+      },
+      outputStrings: tagTeamOutputStrings,
+    },
+    {
+      id: 'R3S Lariat Combo Right Left',
+      type: 'StartsUsing',
+      netRegex: { id: ['9AE8', '9AE9', '9AEA', '9AEB'], source: 'Brute Bomber', capture: true },
```

Comment:
Same request here.  Guessing from context that `9AE8` is a right->right, `9AE9` is a right->left, `9AEA` is left->left, and `9AEB` is left->right, but would be good to document (if that's correct).